qiskit/transpiler/passmanager_config.py

```diff
@@ -124,3 +124,24 @@ def from_backend(cls, backend, **pass_manager_options):
                 res.target = backend.target
 
         return res
+
+    def __str__(self):
+        return (
+            "Pass Manager Config:\n"
```

Comment:
I don't feel very strongly about this at all, but is it worth formatting it as
```python
f"""
PassManagerConfig(
    initial_layout={self.initial_layout},
    basis_gates={self.basis_gates},
...
)
"""
```
so it can easily be copy-paste `eval`ed back into an interpreter?

Reply:
I'm not sure if it's useful to do that in this case because this is not actually going to be a repr string in this case. You won't be able to eval the output because several of the fields are more complex objects that have human readable outputs (`InstructionScheduleMap`, `InstructionDurations`, `CouplingMap`, `Target`, etc)

I personally was just looking for something that I could use for quick print debugging in #7789 that gave me output I could read to show what was getting passed to the `PassManagerConfig`

Reply:
I'm fine with this - what I was suggesting is more like for `repr` anyway.